src/test/java/lotto/model/LottoResultTest.java

```diff
@@ -0,0 +1,38 @@
+package lotto.model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LottoResultTest {
+    @Test
+    @DisplayName("2등 당첨 번호를 비교한다")
+    void matchNumber() {
+        Lotto lotto = new Lotto(Arrays.asList(1, 2, 3, 4, 5, 6));
```

Comment:
```suggestion
        Lotto lotto = new Lotto(List.of(1, 2, 3, 4, 5, 6));
```

위와 같은 정적 팩터리 메서드를 사용해서 구현할 수도 있죠!

봄이 사용한 방식과 제가 제안한 방식의 차이를 학습한 후에 댓글로 남겨주세요 😊

Reply:
### 둘의 공통점
Arrays.asList(), List.of() 모두 사용하면 고정된 크기의 리스트를 반환해주기때문에 add, remove 등 리스트의 개수가 바뀌는 함수 사용이 불가능합니다. (UnsupportedOperationException이 발생합니다.)

### 둘의 차이점
Arrays.asList()는 set() 메서드를 통해 원소의 값이 바뀔 수 있고, null값을 list 내에 갖고, contains로 확인할 수 있습니다.
그리고 참조한 원본 배열의 값이 바뀌면 List의 값도 바뀌고, List의 값이 바뀌면 원본 배열의 값도 바뀝니다.
List.of()는 set() 메서드를 통해 원소의 값이 바뀔 수 없습니다.(UnsupportedOperationException 발생) null값을 list내에 가질 수 없고, contains로 null을 확인할 수 없습니다. 참조한 원본 배열의 값이 바뀌어도 List의 값은 바뀌지 않습니다.

이해한 바를 정리하자면, `Arrays.asList()는 고정된 크기의 list를 return해주어 더이상의 원소를 추가할 수도, 삭제할 수도 없지만, List.of()는 여기서 더 나아가 set() 메서드를 통해 기존의 값도 변경할 수 없고, 원본 배열의 값이 바뀌어도 바뀌지 않는 점에서 불변이라고 할 수 있다.` 입니다.

<br/>
공부한 내용 첨부합니다. 잘못된 부분이 있다면 알려주세요🥺
https://www.notion.so/Arrays-asList-vs-List-of-6ad225aa29d346c39e6ed8bbf8863277

Reply:
크흐 너무 잘 정리해주셨네요!! 제가 질문한 두가지 방법 이외의 다른 배열 생성에 대해서도 정리하신 점이 인상깊었습니다!
💯